sentry-ruby/lib/sentry/transport.rb

```diff
@@ -5,16 +5,24 @@ module Sentry
   class Transport
     PROTOCOL_VERSION = '7'
     USER_AGENT = "sentry-ruby/#{Sentry::VERSION}"
+    CLIENT_REPORT_INTERVAL = 60
 
     include LoggingHelper
 
+    attr_accessor :configuration
     attr_reader :logger, :rate_limits
 
     def initialize(configuration)
+      @configuration = configuration
       @logger = configuration.logger
       @transport_configuration = configuration.transport
       @dsn = configuration.dsn
       @rate_limits = {}
+
+      if configuration.send_client_reports
+        @discarded_events = Hash.new(0)
```

Comment:
is this different than direct hash declaration `{}`?

Reply:
it sets a default value of 0 so you don't have to do `@hash[key] ||= 0` every time.

Reply:
ah so it's like passing the default proc. thanks!